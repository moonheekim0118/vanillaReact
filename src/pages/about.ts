
 const About=()=>{

    const container = document.createElement('div');
    container.className='container';
    container.innerText='about page';
    
    return container;
}

export default About();
