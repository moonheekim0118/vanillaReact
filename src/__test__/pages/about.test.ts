import "jsdom-global/register";
import about from "../../pages/about";

describe("about page rendering", () => {
    let container;

    it("renders correctly", () => {
        container = about();
    });

    it("matches snapshot", () => {
        expect(container).toMatchSnapshot();
    });
});
