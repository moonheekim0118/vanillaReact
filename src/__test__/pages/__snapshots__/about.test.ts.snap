// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page rendering matches snapshot 1`] = `
<side
  class="container"
>
  
        
  <h2>
    a Swiftie made this site just for funðŸ¤©
  </h2>
  
        
  <p>
    I use 
    <a
      href="https://taylor.rest/"
    >
      taylor rest API
    </a>
     for serving her photos and quotes
  </p>
  
        
</side>
`;
